src/toil_scripts/tools/preprocessing.py

```diff
@@ -211,16 +365,17 @@ def run_indel_realignment(job, intervals, bam, bai, ref, ref_dict, fai, phase, m
     if unsafe:
         parameters.extend(['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'])
     docker_call(tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
-                inputs=inputs,
+                inputs=inputs, mock=mock,
                 outputs={'sample.indel.bam': None, 'sample.indel.bai': None},
-                work_dir=work_dir, parameters=parameters, env=dict(JAVA_OPTS='-Xmx{}'.format(mem)))
+                work_dir=work_dir, parameters=parameters, env=dict(_JAVA_OPTIONS='-Djava.io.tmpdir=/data/ -Xmx{}'.format(mem)))
     # Write to fileStore
     indel_bam = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'sample.indel.bam'))
     indel_bai = job.fileStore.writeGlobalFile(os.path.join(work_dir, 'sample.indel.bai'))
     return indel_bam, indel_bai
 
 
-def run_base_recalibration(job, cores, indel_bam, indel_bai, ref, ref_dict, fai, dbsnp, mem, unsafe=False):
+def run_base_recalibration(job, cores, indel_bam, indel_bai, ref, ref_dict, fai, dbsnp, mills,
+                           mem='10G', bqsr=None, unsafe=False, mock=False):
```

Comment:
Document `mock`. Split params out to one per line unless there is a logical grouping.